source/mongodump.txt

```diff
@@ -816,6 +833,13 @@ Options
    You cannot use the :option:`--archive <mongodump --archive>` option
    with the :option:`--out <mongodump --out>` option.
 
+   .. note::
+
+      :binary:`~bin.mongodump` performance may improve if the process
```

Comment:
[tiny nit] [style guide](https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#use-modal-verbs-accurately) - "can" might be a better suited modal verb for this instance 
```suggestion
      :binary:`~bin.mongodump` performance can improve if the process
```

Reply:
Done!